stdlib/tkinter/__init__.pyi

```diff
@@ -2891,6 +2891,7 @@ class Scrollbar(Widget):
     def set(self, first: float | str, last: float | str) -> None: ...
 
 _TextIndex: TypeAlias = _tkinter.Tcl_Obj | str | float | Misc
+_TextChildAlign: TypeAlias = Literal["top", "center", "bottom", "baseline"]
```

Comment:
I would actually prefer copy/pasting `Literal["top", "center", "bottom", "baseline"]` into all places where it is used. Type aliases make stubs more confusing, and also provide a worse experience for IDE users: you see `_TextChildAlign` instead of the list of possible values.